tracing/tracing-zipkin/src/test/java/org/apache/servicecomb/tracing/zipkin/ZipkinTracingAdviserTest.java

```diff
@@ -134,11 +146,11 @@ public void startsNewChildSpan() {
 
     MatcherAssert.assertThat(traces.size(), is(nThreads));
 
-    for (Queue<zipkin2.Span> queue : traces.values()) {
-      zipkin2.Span child = queue.poll();
+    for (Queue<MutableSpan> queue : traces.values()) {
+      MutableSpan parent = queue.poll();
```

Comment:
Why change the `poll` order, is this change compatible?

Reply:
repaired,